src/api/repository/mongo.py

```diff
@@ -0,0 +1,2 @@
+def save_articles(mongo, data):
```

Comment:
replace `data` with `article` which is a class of Article (does not have to be necessary an Article Model, but rather some data structure, ie. ./data/article.py)

Reply:
OK